cpp/CMakeLists.txt

```
@@ -129,7 +133,7 @@ target_include_directories(
                    "$<INSTALL_INTERFACE:include>"
 )
 target_link_libraries(
-  kvikio INTERFACE Threads::Threads ${CMAKE_DL_LIBS} nvtx3::nvtx3-cpp BS::thread_pool
+  kvikio INTERFACE Threads::Threads BS::thread_pool ${CMAKE_DL_LIBS} ${NVTX3_LIBRARIES}
```

```suggestion
  kvikio INTERFACE Threads::Threads BS::thread_pool ${CMAKE_DL_LIBS} $<TARGET_NAME_IF_EXISTS:nvtx3::nvtx3-cpp>
```
Slightly cleaner, and doesn't rely on `NVTX3_LIBRARIES` possibly being empty.